src/test/scala/special/sigma/SigmaDslTest.scala

```diff
@@ -426,4 +426,16 @@ class SigmaDslTest extends PropSpec
     }
 
   }
+
+  property("Option methods equivalence") {
+    val opt: Option[Long] = ctx.dataInputs(0).R0[Long]
+    val eq = EqualityChecker(opt)
+    eq({ (x: Option[Long]) => x.get })("{ (x: Option[Long]) => x.get }")
+    // TODO implement Option.isEmpty
+    //  eq({ (x: Option[Long]) => x.isEmpty })("{ (x: Option[Long]) => x.isEmpty }")
+    eq({ (x: Option[Long]) => x.isDefined })("{ (x: Option[Long]) => x.isDefined }")
+    eq({ (x: Option[Long]) => x.getOrElse(1L) })("{ (x: Option[Long]) => x.getOrElse(1L) }")
+    eq({ (x: Option[Long]) => x.filter(_ == 1) })("{ (x: Option[Long]) => x.filter({ (v: Long) => v == 1 }) }")
```

Comment:
@greenhat yes, we can write shorter code in Scala, but here the test should prove equivalence of Scala and ErgoScript. If the code is different, it actually doesn't prove such an equivalence.

Reply:
Got it. Fixed.